### ISSUE TITLE
Ping an/ausschalten

### ISSUE THREAD

**Issue by Cautex** [Floxiii/Lobby#4]
Ich würde mir wünschen, dass man das Ping von der Lobby ausschalten kann.

**Comment by Floxiii** [comment]
Meinst du den Befehl?

**Comment by Cautex** [comment]
Ja, also dass wenn man /ping eingibt dass da nicht das vom Lobby Sytem kommt. Weil ich will auf allen servern ja das Gleiche ping zeichen und ist kaka wenn das auf der Lobby anders ist
